build.gradle

```diff
@@ -1,5 +1,5 @@
 plugins {
-    id 'play'
+    id 'org.gradle.playframework' version '0.6'
     id 'base'
```

Comment:
Hi,
I don't think that we need the `base` plugin.

Reply:
yes it's removed